olive/model/hf_utils.py

```diff
@@ -207,6 +207,7 @@ class HFConfig(ConfigBase):
     # feature is optional if task is specified and don't need past
     # else, provide feature such as "causal-lm-with-past"
     feature: str = None
+    token: str = None
```

Comment:
I am not sure it is right approach. But my gut feeling to keep a text token is not a good practice.